native/screens/UserProfile.jsx

```diff
@@ -101,33 +111,37 @@ export const UserProfile = ({ route, navigation }) => {
           setReviewStatus={setReviewStatus}
         />
       )}
-      <View>
-        <Text>
+      <View style={styles.nameContainer}>
+      <View style={styles.placeholderImage}></View>
+        <Text style={styles.name}>
           {first_name} {last_name}
         </Text>
         <Text>{role}</Text>
       </View>
 
-      <Text> I AM AN IMAGE</Text>
 
-      <View>
-        <Text>{email}</Text>
-        <Text>PORTFOLIO LINK</Text>
+      <View style={styles.wordContainer}>
+        <Text style={styles.title}>ABOUT</Text>
+        <Text style={styles.words}>{about}</Text>
       </View>
 
-      <View>
-        <Text>{about}</Text>
+      <View style={styles.wordContainer}>
+      <Text style={styles.title}>FUN FACT</Text>
+        <Text style={styles.words}>{fun_fact}</Text>
       </View>
 
-      <View>
-        <Text>{fun_fact}</Text>
+      <View style={styles.wordContainer}>
+        <Text style={styles.portfolio }
+      onPress={() => Linking.openURL(`${portfolio_link}`)}> Portfolio </Text>
```

Comment:
this is curious! I've never seen a text with an onpress or the `Linking` function... very cool!

Reply:
On a similar and this is not your fault, I don't know why there was a box for the Portfolio URL here on the figma. I don't think a simple link merits a whole box and I feel like it would be better located at the top of the screen near the user's name or something. 

I'm going to bring this up to the UX team but we can delete this for now, I don't think it will survive that discussion